homeassistant/components/hunterdouglas_powerview/config_flow.py

```diff
@@ -51,6 +63,11 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    @staticmethod
+    def async_get_options_flow(config_entry):
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
```

Comment:
We do require 100% coverage for the options flows, but per my other comment, I don't think its needed.